versioned_docs/version-2.18/reference/subsystems/nodejs-infer.mdx

```diff
@@ -11,7 +11,7 @@ import styles from "@site/src/components/reference/styles.module.css";
 
 Options controlling which dependencies will be inferred for javascript targets.
 
-Backend: <span className="color--primary">`pants.backend.experimental.openapi.lint.spectral`</span>
+Backend: <span className="color--primary">`pants.backend.experimental.javascript`</span>
```

Comment:
It is a a list in the raw data.. However, for `environments-preview` I don't see the relevance.. well, unless we want to present all backends _using_ it...

Reply:
So this comes from the `provider` field in the `help-all` JSON which is a string.

```
josh@cephandrius:~/work/pantsbuild.org$ jq '.scope_to_help_info["environments-preview"].provider' docs/reference/help-all.json
```

I'm wondering if that ought to be a list (or a different field, to preserve backwards compatibility) of which backends provides this subsystem. E.g. for this one, `nodejs-infer` it will exist if you depend on _any_ of the JS backends.

Reply:
Yea, the raw data is a list, turned into a single string value by: 
https://github.com/pantsbuild/pants/blob/e7b4b251b74f5e8332e6d527d0c7a5a3ffc878ab/src/python/pants/help/help_info_extracter.py#L646-L651

The idea was to identify the _single_ backend responsible for each option/rule/etc. If we want the full list, I'd suggest having that in a dedicated field, perhaps like "references" or something like that.